packages/engine/Specs/Scene/Model/ModelSpec.js

```diff
@@ -2128,13 +2130,11 @@ describe(
           },
           imageryLayersUpdatedEvent: new Event(),
           destroy: function () {},
+          beginFrame: function () {},
+          endFrame: function () {},
+          terrainProviderChanged: new Event(),
         };
 
-        globe.beginFrame = function () {};
```

Comment:
Should there be a top-level exaggeration spec for both `Model` and `Cesium3dTileset` similar to the one in `packages/engine/Specs/Core/QuantizedMeshTerrainDataSpec.js`?

Reply:
Additionally, one on for `scene` that confirms the properties exist and can be get/set?